pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
+> More information: <https://rmlint.readthedocs.io/en/latest/index.html>.
```

Comment:
```suggestion
> More information: <https://rmlint.readthedocs.io/en/latest/rmlint.1.html>.
```
My suggestion would be to directly link to the man page.

Reply:
I'd normally prefer the main website when it does contain actual information (as opposed to many project home pages nowadays that seem to me little more than an interactive advertisement poster), which is the case here; however, the manpage is also very well written, and more to-the-point regarding how to use the tool, so in this case I'll agree with @kbdharun that it's a better target for the More info link.

Reply:
Applied in 40d9eedce1fd7c86defad42ecef67722ff95a17c.